geolocation/ios/Plugin/GeolocationPlugin.swift

```diff
@@ -0,0 +1,200 @@
+import Foundation
+import CoreLocation
+import UIKit
+import Capacitor
+
+public struct GeolocationCoords {
+    public var latitude: Double
+    public var longitude: Double
+    public init(latitude: Double, longitude: Double) {
+        self.latitude = latitude
+        self.longitude = longitude
+    }
+}
+
+@objc(GeolocationPlugin)
+public class GeolocationPlugin: CAPPlugin, CLLocationManagerDelegate {
+
+    enum CallType {
+        case permissions
+        case singleUpdate
+        case watch
+    }
+
+    var locationManager = CLLocationManager()
+    private var callQueue: [String: CallType] = [:]
+
+    @objc func getCurrentPosition(_ call: CAPPluginCall) {
+        call.save()
+        callQueue[call.callbackId] = .singleUpdate
+
+        DispatchQueue.main.async {
+            self.locationManager.delegate = self
+            self.locationManager.requestWhenInUseAuthorization()
+
+            if call.getBool("enableHighAccuracy", false) == true {
+                self.locationManager.desiredAccuracy = kCLLocationAccuracyBest
+            } else {
+                self.locationManager.desiredAccuracy = kCLLocationAccuracyThreeKilometers
+            }
+
+            self.locationManager.requestLocation()
+        }
+    }
+
+    @objc func watchPosition(_ call: CAPPluginCall) {
+        call.save()
+        callQueue[call.callbackId] = .watch
+
+        DispatchQueue.main.async {
+            self.locationManager.delegate = self
+            self.locationManager.requestWhenInUseAuthorization()
+
+            if call.getBool("enableHighAccuracy", false) == true {
+                self.locationManager.desiredAccuracy = kCLLocationAccuracyBestForNavigation
+            } else {
+                self.locationManager.desiredAccuracy = kCLLocationAccuracyThreeKilometers
+            }
+
+            self.locationManager.startUpdatingLocation()
+        }
+    }
+
+    @objc func clearWatch(_ call: CAPPluginCall) {
+        guard let callbackId = call.getString("id") else {
+            CAPLog.print("Must supply id")
+            return
+        }
+        let savedCall = bridge?.getSavedCall(callbackId)
+        if savedCall != nil {
+            bridge?.releaseCall(savedCall!)
```

Comment:
The check can be combined into a single line to remove the force unwrap.
```suggestion
        if let savedCall = bridge?.getSavedCall(callbackId) {
            bridge?.releaseCall(savedCall)
```